components/selectDevice/deviceList.tsx

```diff
@@ -0,0 +1,25 @@
+import Device from "./device";
+
+const DeviceList = ({
+  deviceList,
+  currentDevice,
+}: {
+  deviceList: SpotifyApi.UserDevice[];
+  currentDevice: SpotifyApi.CurrentPlaybackResponse | undefined;
+}) => (
+  <>
+    <h3 className="text-lg font-bold">Select a device</h3>
+    <div className="flex flex-col space-y-2">
```

Comment:
instead of making this a flex, use a `btn-group btn-vertical` I think it will look better